public/index.php

```diff
@@ -25,12 +25,16 @@
 ini_set("display_errors", 1);
 session_start();
 header('Content-Type: text/html; charset=utf-8');
+//Sanitize $_POST parameters
+$post_input  = filter_input_array(INPUT_POST, FILTER_SANITIZE_STRING);
 
 $templates = new League\Plates\Engine('../templates');
 
+$post_input['oauth_consumer_key'] = filter_input(INPUT_POST,'oauth_consumer_key',FILTER_SANITIZE_NUMBER_INT);
```

Comment:
I'm falling back to FILTER_SANITIZE_STRING for now.  I know with LTI tools the consumer keys are pasted in by the user - so if that's the case here, they'll be just about anything.